### ISSUE TITLE
Account Does Not Exist or Account Balance is Too Low

### ISSUE THREAD

**Issue by sparcusa** [ethereum/mist#1131]
#### System information
``````
Version: `0.8.2`
OS & Version: windows 7
Node type: `geth 1.4.1`
``````
Sorry of this is the wrong forum for this 

![image](https://cloud.githubusercontent.com/assets/21336510/18089504/4cd4a6bc-6e8e-11e6-8508-b3f5e4f097de.png)

I am working through the ethereum.org DApp tutorial on TESTNET when I get to the contract deployment stage and I get this error "Account Does Not Exist or Account Balance is Too Low".   When I attempt to copy the account address to send Eth to that account I get this message

![image](https://cloud.githubusercontent.com/assets/21336510/18089408/d9b4056a-6e8d-11e6-8d59-2e4a35b6c2ad.png)

So how do you deploy a contract on the test net if you can't send ETH to the account?

**Comment by luclu** [comment]
> Sorry of this is the wrong forum for this 

You are spot on here (and also on [gitter](https://gitter.im/ethereum/tutorials)) ;)

> [...] and I get this error "Account Does Not Exist or Account Balance is Too Low".

This means that you will need test Ether to interact (write/change state) with the blockchain. Head over to https://zerogox.com/ethereum/wei_faucet.

> When I attempt to copy the account address to send Eth to that account I get this message

This is just a precaution to prevent mixing up main-net and test-net addresses.

> So how do you deploy a contract on the test net if you can't send ETH to the account?

Make sure you got the conceptional difference between accounts and contracts. Head to http://ethdocs.org/ for that.

**Comment by luclu** [comment]
I will close this for now as of inactivity. Feel free to stick around and ask further questions though.

**Comment by gitgong** [comment]
@luclu  Same symptom/problem as above  doing the currency contract tutorial, when I go to deploy the contract I get the message: 

Account Does Not Exist or Account Balance is Too Low".
I have 575 ether (the free test ones), which I mined in the test-net environment, listed in the from account. TEST-NET is clearly shown in the bar.

Everything looks proper as the tutorial, except the pragma version was missed. 

Since I started loading the blockchain, I have 6 peers and 43% (985,000 blocks left) showing in my block chain download bar. Do I need to wait for this to complete before testing, or am I missing something?
